doc/developers/advanced_installation.rst

```diff
@@ -28,7 +28,7 @@ Installing a nightly build is the quickest way to:
 
 .. prompt:: bash $
 
-  pip install --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn
+  pip install --upgrade --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn
```

Comment:
Here's a combination that seems to work as expected (found here https://stackoverflow.com/questions/19548957/can-i-force-pip-to-reinstall-the-current-version)
```suggestion
  pip install --force-reinstall --no-deps --pre --extra-index https://pypi.anaconda.org/scipy-wheels-nightly/simple scikit-learn
```